charts/pokeshop-demo/values.yaml

```diff
@@ -33,7 +36,7 @@ pokeshop:
     - name: COLLECTOR_ENDPOINT
       value: http://ttdemo-opentelemetry-collector:44317
     - name: DATABASE_URL
-      value: postgresql://ashketchum:squirtle123@postgresql:5432/pokeshop?schema=public
```

Comment:
`schema=public` is the default and it breaks `psql` for some reason, so I removed it.